src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/proteins-list-item-info/ProteinsListItemInfo.tsx

```diff
@@ -93,6 +91,10 @@ const ProteinsListItemInfo = (props: Props) => {
   const { product } =
     transcriptWithProteinDomains?.product_generating_contexts[0] || {};
 
+  const uniprotXref = product?.external_references.find(
+    (xref) => xref.source.id == SWISSPROT_SOURCE
```

Comment:
I failed to notice this before approving. We need to use triple equals (`===`) for conditionals

Reply:
@imransl would you like to add an eslint rule for this? 😃 

https://eslint.org/docs/rules/eqeqeq